src/map/conquest_system.h

```diff
@@ -44,7 +44,7 @@ namespace conquest
 {
     std::shared_ptr<ConquestData> GetConquestData(); // Cached data with influences / region controls
 
-    void HandleZMQMessage(uint8* data);
+    void HandleZMQMessage(uint8* data); // Called whenever a ZMQ message is recieved from world server
```

Comment:
Should there be a `RegionHandlerClient` base class or something, an equivalent to the message handlers and systems in world, but for client?

Reply:
Actually I think I will do that change in a separate PR, before merging back to LSB. The change affects conquest system too which is out of the scope of this PR.